card/lib/card/format.rb

```diff
@@ -392,9 +392,12 @@ def prepare_nest opts
 
       opts ||= {}
       case
-      when opts.has_key?( :comment )                            ; opts[:comment]   # as in commented code
-      when @mode == :closed && @char_count > Card.config.max_char_count   ; ''     # already out of view
-      when opts[:inc_name]=='_main' && !Env.ajax? && @depth==0  ; expand_main opts
+      when opts.has_key?( :comment )                                      ; opts[:comment]
+        # commented nest
+      when @mode == :closed && @char_count > Card.config.max_char_count   ; ''
+        # already out of view
+      when opts[:inc_name]=='_main' && show_layout? && @depth==0          ; expand_main opts
+        # why do we have these restrictions on main?
```

Comment:
The sense of it seems to be only doing this in a layout rule card.  This is the only place the _main is handled, right?.

Maybe it is for viewing layout cards, and not looping the inclusion?